### PR TITLE
Fixed padding on forms on large screens

### PR DESCRIPTION
**Pull request by Porim**
Hey Nik, I've fixed that issue you recently raised about padding on larger screens on pages with forms. Please test it out and merge it if you are happy with the way it works now. If not happy, let me know what's wrong. Thanks.

### CODE REVIEW

File: app/assets/stylesheets/components/_footer.scss [hunk 1]
```diff
@@ -19,7 +19,7 @@
   height: 100%;
   justify-content: space-between;
   padding-bottom: 10px;
-  padding-top: 23px;
```
Comment (nikkegg) [inline app/assets/stylesheets/components/_footer.scss:22]: i think perhaps padding-top: 23px should be brought back? I attached screenshoot in the comments, you can see white space between footer and form on 13inch screen, but only very narrow one.

**Review by nikkegg — CHANGES_REQUESTED** [review]

**Comment by nikkegg** [comment]
i attach screenshots with the white space from 13inch mac
Screenshot 2020-11-02 at 19 31 52

File: app/assets/stylesheets/components/_footer.scss (hunk 1 above)
Comment (Porim) [reply app/assets/stylesheets/components/_footer.scss:22]: I have tested it pixel by pixel and 18px is the only value that fitted perfectly. I have tested it in various sizes on my desktop PC and also tested in on my 13" XPS, but I can't recreate the white space effect that you experiencing on your Mac 13. Anything more than 18px adds scrolling to the page.

Comment (Porim) [reply app/assets/stylesheets/components/_footer.scss:22]: So whatever is the solution to the mentioned issue, it's not this padding.

**Pull request merged**